pyspark/bigdl/util/common.py

```diff
@@ -434,6 +434,18 @@ def _test():
     if failure_count:
         exit(-1)
 
+class TransformationError(BaseException):
+    pass
+
+def ndarray_to_rdd(X, y):
+    if type(X) != np.ndarray or type(y) != np.ndarray:
+        raise TransformationError('Input arrays should be numpy ndarrays')
```

Comment:
I think simply use `Exception` is fine here as we haven't have uniform customized exception for now.